cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,86 @@
+= CobiGen Angular client generation
+====
+With the CobiGen code generator you can easily generate components for your angular application.
+The following tutorial provides an example for using the CobiGen cli.
+You can find more information on https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
+== Prerequisites
+* devonfw Ide
+* Install yarn globally
+== Learning goals.
+Here in this tutorial we will learn the following:
+* Create an angular project
+* Cobigen to generate the angular source code out of the java files
+* Change the user interface in the angular project to show the new components
+* Show the newly created components
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn", "node", "ng", "npm"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you. The next step is to install the CobiGen CLI.
+[step]
+--
+installCobiGen()
+--
+
+CobiGen uses the backend classes for generating your code used in the angular application. So in the next steps you will create a java project, create a simple java class and run the backend server.
+[step]
+--
+createDevon4jProject("cobigenexample")
+--
+
+====
+[step]
+== Build and Run Java Server
+--
+buildJava("cobigenexample")
+runServerJava("cobigenexample/server",{ "startupTime": 180, "port": 8080, "path": "cobigenexample" })
+--
+Now the Java Server should be running.
+====
+
+CobiGen is integrated via plugin in the VS Code IDE. We will use it to generate code from one single java class based on existing templates.
+[step]
+--
+createDevon4ngProject("cobigenexample", "")
```

Comment:
You are trying to create a angular project with the name cobigenexample, but you already, created a project in the same directory with the same name